test/styles/bibtex.js

```diff
@@ -0,0 +1,72 @@
+const assert = require('assert');
+const bibtexmisc = require('../../model/styles/bibtexMisc');
+const biblatexSoftware = require('../../model/styles/biblatexSoftware');
+
+const oneAuthorSourceData = require('../mockSourceData/allFieldsOneAuthor');
+const threeAuthorsSourceData = require('../mockSourceData/allFieldsThreeAuthors');
+
+describe('BibTexMiscStyle', () => {
+  describe('format', () => {
+    it('Month field, one author', () => {
+      let formattedString = bibtexmisc.format(oneAuthorSourceData);
```

Comment:
Because you have multiple tests for the one author source data we should only call the format method once.  I think something like this might work:

```javascript 
describe('format', () => {
  describe('One Author', () => {
    const formattedString = bibtexmisc.format(oneAuthorSourceData);

    it('Month', () => {
      //Tests code here
    }

    it('Title', () =>() {
      // Test code here
    }
  }
}
```